src/pages/plugins/concepts/manifest/index.md

```diff
@@ -0,0 +1,863 @@
+                <li>pluginList: suitable for display in a plugin list. Icon sizes are 24x24px for 100% scaling and 48x48px for 200% scaling.</li>
+            </ul>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>["generic"]</inlineCode> (suitable for display anywhere)</p>
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+### EntrypointDefinition
+
+<p>Represents an entrypoint provided by the plugin, which can be invoked by the user.</p>
+<p>An entrypoint consists of an ID and a label at minimum.</p>
+<p><strong>Example</strong></p>
+<code class="language-json">{`{
+    "id": "myPlugin.myEntrypoint",
+    "label": "My Entrypoint"
+}`}</code>
+<h4>Properties</h4>
+<table >
+    <thead>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><inlineCode>type</inlineCode> *</td>
+        <td><inlineCode>"command" | "panel"</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The type of entrypoint. Currently, only <inlineCode>command</inlineCode> and <inlineCode>panel</inlineCode> are supported.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>id</inlineCode> *</td>
+        <td><inlineCode>string</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The ID of the entrypoint. This ID must be unique within the plugin. It is used to identify the code that implements the entrypoint.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>label</inlineCode> *</td>
+        <td><inlineCode>string | Localized<wbr />String</inlineCode></td>
+        <td>required</td>
+        <td>
+            <p>The label of the entrypoint. This label is used to display the entrypoint to the user, such as in the plugin menu.</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>description</inlineCode></td>
+        <td><inlineCode>string | Localized<wbr />String</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>A description of the entrypoint. This description is used in tooltips and other places where a longer description is appropriate, depending on the host app.</p>
+            <p><strong>Default value</strong></p>
+            <p><inlineCode>undefined</inlineCode> (use plugin name)</p>
+        </td>
+    </tr>
+    <tr>
+        <td><inlineCode>shortcut</inlineCode></td>
+        <td><inlineCode>{'{'} mac: string, win: string {'}'}</inlineCode></td>
+        <td>optional</td>
+        <td>
+            <p>A keyboard shortcut that can be used to invoke the entrypoint.</p>
+            <p>Keyboard shortcuts are specified separately for Windows and macOS platforms. If the shortcut is not available in the host application, it will be ignored.</p>
+            <p><strong>Remarks</strong></p>
+            <p>Currently, keyboard shortcuts are supported in Adobe XD only.</p>
```

Comment:
If its only XD, let's remove it.

Reply:
@anoopvalomkot Please verify

Reply:
Let's not remove it. It's a part of UXP (since it's at least in one app). So if we omit it, this will just cause confusion for developers.

Reply:
@padmkris123 I believe this needs development on InDesign side and hence is not supported today. But this could be a good feature to add later, we can expect customers to ask for it.